Makefile

```diff
@@ -48,7 +48,7 @@ dep:
 	cp -Rp build/rollup build/electron-app/app
 	cp -Rp build/rollup/resources build/electron-app
 	cp -Rp build/rollup/manifest build/electron-app
-	sed -i -E 's/\.\/dist\/components\/backend-ai-webui.js/es6:\/\dist\/components\/backend-ai-webui.js/g' build/electron-app/app/index.html
+	sed -i -E 's/\.\/dist\/components\/backend-ai-webui.js/es6:\/\/dist\/components\/backend-ai-webui.js/g' build/electron-app/app/index.html
```

Comment:
`es6:\/\dist\/` is missing `/`, right? 😁  I just added one more `/` here.  now, `es6:\/\/dist\/`. @inureyes